src/main/java/com/google/javascript/clutz/ClutzErrorManager.java

```diff
@@ -0,0 +1,49 @@
+package com.google.javascript.clutz;
+
+import com.google.javascript.jscomp.CheckLevel;
+import com.google.javascript.jscomp.JSError;
+import com.google.javascript.jscomp.MessageFormatter;
+import com.google.javascript.jscomp.PrintStreamErrorManager;
+
+import java.io.PrintStream;
+
+/**
+ * An error manager that delays reporting though {@link #doGenerateReport()}, allows failing fast,
+ * and that adds special, Clutz-specific errors.
+ */
+final class ClutzErrorManager extends PrintStreamErrorManager {
+  private final boolean debug;
+  boolean reportClutzMissingTypes = true;
+
+  /**
+   * @param failEarly if true, directly fail on the first reported error (but not warning!).
```

Comment:
Yeah, removed it again as it doesn't seem to be useful enough. So far it has been easy enough to reduce issues to small reproducible cases and attach a debugger to it, so no real need to run it in "throw a stack trace" mode.